docs/advanced.rst

```diff
@@ -247,52 +247,149 @@ Anaconda uses Conda to manage packages. To reuse Conda–installed Python packag
 
 Sometimes, you would want to generate a requirements file based on your current
 environment, for example to include tooling that only supports requirements.txt.
-You can convert a ``Pipfile`` and ``Pipfile.lock`` into a ``requirements.txt``
+You can convert a ``Pipfile.lock`` into a ``requirements.txt``
 file very easily.
 
 Let's take this ``Pipfile``::
 
     [[source]]
     url = "https://pypi.python.org/simple"
     verify_ssl = true
+    name = "pypi"
 
     [packages]
-    requests = {version="*"}
+    requests = {version="==2.18.4"}
 
     [dev-packages]
-    pytest = {version="*"}
+    pytest = {version="==3.2.3"}
+
+Which generated the following ``Pipfile.lock`` upon completion of running ``pipenv lock```::
+
+    {
+		"_meta": {
+			"hash": {
+				"sha256": "4b81df812babd4e54ba5a4086714d7d303c1c3f00d725c76e38dd58cbd360f4e"
+			},
+			"pipfile-spec": 6,
+			"requires": {},
+			"sources": [
+				{
+					"name": "pypi",
+					"url": "https://pypi.python.org/simple",
+					"verify_ssl": true
+				}
+			]
+		},
+		"default": {
+			"certifi": {
+				"hashes": [
+					"sha256:0d9c601124e5a6ba9712dbc60d9c53c21e34f5f641fe83002317394311bdce14",
+					"sha256:90c1a32f1d68f940488354e36370f6cca89f0f106db09518524c88d6ed83f382"
+				],
+				"markers": "python_version >= '3.6'",
+				"version": "==2022.9.24"
+			},
+			"chardet": {
+				"hashes": [
+					"sha256:84ab92ed1c4d4f16916e05906b6b75a6c0fb5db821cc65e70cbd64a3e2a5eaae",
```

Comment:
I think you can trim part of the contents of the lock file from here to the develop section, and then remove also parts of the develop section.

This will be more readable IMHO.